airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py

```diff
@@ -69,26 +69,32 @@ def build_container(
     return container
 
 
+def warmup_directory(dagger_client: dagger.Client, formatter: Formatter) -> Union[dagger.Directory, None]:
+    """Creates a Dagger directory which includes the warmup files for the given Formatter."""
+    include = WARM_UP_INCLUSIONS.get(formatter)
+    if not include:
+        return None
+    include = include + [f"{AIRBYTE_SUBMODULE_DIR_NAME}/{i}" for i in include]
```

Comment:
Shouldn't we check thaht `AIRBYTE_SUBMODULE_DIR_NAME` exists first? I'm not sure if dagger fails if you tell it to include not existing path (as the submodule does not exists in the main repo)

Reply:
No, conveniently enough, it's fine if the path doesn't exist!